wordpress-mobile/WordPress-iOS
# Update release-toolkit

This PR updates the `release-toolkit` to fix an issue we found here: https://github.com/wordpress-mobile/WordPress-iOS/pull/12192.

The PR is based on the changes here: https://github.com/wordpress-mobile/release-toolkit/pull/67

### To test:
1. `rm -rf vendor/ && bundle install`.
2. Run an internal build and verify that it works. 

### Note:
A new version of  `release-toolkit` should be released and the reference in `Pluginfile` updated before merging this. 

Update release notes:
